### PR TITLE
chore(disk-fill): Addding option to specify ephemeral storage(Mibi) e…

### PR DESCRIPTION
**Pull request by ispeakc0de**
Signed-off-by: shubhamchaudhary <shubham@chaosnative.com

- Adding option to specify ephemeral storage(MiBi) explicitly via an env `EPHEMERAL_STORAGE_MEBIBYTES`
-  if both the fill percentage and EPHEMERAL_STORAGE_MEBIBYTES are provided it will prefer fill percentage over it -- given that ephemeral-storage limits are provided inside target application.

Tested for:
- if none of EPHEMERAL_STORAGE_MEBIBYTES and  ephemeral-storage limits inside the target application provided :heavy_check_mark: 
- if only EPHEMERAL_STORAGE_MEBIBYTES provided :heavy_check_mark: 
- if only ephemeral-storage limits  inside the target application provided  :heavy_check_mark: 
 - if both of EPHEMERAL_STORAGE_MEBIBYTES and  ephemeral-storage limits inside the target application provided :heavy_check_mark:

### CODE REVIEW

**Comment by ispeakc0de** [comment]
### helper logs

```yaml
time="2021-03-10T08:18:58Z" level=info msg="[PreReq]: Getting the ENV variables"
time="2021-03-10T08:18:58Z" level=info msg="used ephemeral storage space: 28"
time="2021-03-10T08:18:58Z" level=info msg="[Info]: Details of application under chaos injection" ContainerID=637c57948c8d9d4469b2680270398e2058edf6fc8f0fbfcfcabe317791b8e859 ContainerName=nginx PodName=nginx-595b696f9b-b8m89 ephemeralStorageLimit=4294968
time="2021-03-10T08:18:58Z" level=info msg="ephemeral storage size to be filled: 3435946"
time="2021-03-10T08:19:22Z" level=info msg="[Chaos]: Waiting for 60s"
time="2021-03-10T08:20:22Z" level=info msg="[Chaos]: Time is up for experiment: disk-fill"

```

File: chaoslib/litmus/disk-fill/helper/disk-fill.go [hunk 1]
```diff
@@ -96,9 +83,26 @@ func DiskFill(experimentsDetails *experimentTypes.ExperimentDetails, clients cli
...
+		return errors.Errorf("Either provide ephemeral storage limit inside target container or define EPHEMERAL_STORAGE_MEBIBYTES ENV")
+	}
+
 	// deriving the ephemeral storage size to be filled
 	sizeTobeFilled := GetSizeToBeFilled(experimentsDetails, usedEphemeralStorageSize, int(ephemeralStorageLimit))
 
+	log.InfoWithValues("[Info]: Details of application under chaos injection", logrus.Fields{
+		"PodName":               experimentsDetails.TargetPods,
+		"ContainerName":         experimentsDetails.TargetContainer,
+		"ephemeralStorageLimit": ephemeralStorageLimit,
```
Comment (uditgaurav) [inline chaoslib/litmus/disk-fill/helper/disk-fill.go:102]: What is the limit of ephemeralStorageLimit we get can we mention it ...  also in line 106

Comment (ispeakc0de) [reply chaoslib/litmus/disk-fill/helper/disk-fill.go:102]: didn't get it. Can you please elaborate?

Comment (uditgaurav) [reply chaoslib/litmus/disk-fill/helper/disk-fill.go:102]: unit*

**Review by ksatchit — APPROVED** [review]

**Review by uditgaurav — APPROVED** [review]

**Review by ksatchit — APPROVED** [review]

**Pull request merged**